app/views/idv/gpo_verify/index.html.erb

```diff
@@ -9,16 +9,18 @@
 
 <% title t('forms.verify_profile.title') %>
 
-<%= render AlertComponent.new(type: :info, class: 'margin-bottom-4', text_tag: 'div') do %>
-  <p>
-    <%= t('forms.verify_profile.alert_info') %>
-    <br>
-    <%= render 'shared/address', address: @gpo_verify_form.pii %>
-  </p>
-  <p>
-    <%= t('forms.verify_profile.wrong_address') %>
-    <%= link_to t('forms.verify_profile.clear_and_start_over'), idv_confirm_start_over_path %>
-  </p>
+<% if @gpo_verify_form.pii %>
```

Comment:
should we add an `else` here or maybe something in the controller that checks, and does like a `Rails.logger.warn` so we can track how often this happens? Maybe an analytics event so it's tied to user ID?

Reply:
Yeah I like the analytics event idea, will add